### DISCUSSION TITLE
Downloaded video and audio as separate files

### DISCUSSION THREAD

**Discussion by fluger12** [shaked6540/YoutubePlaylistDownloader#180] [Q&A]
Just installed and downloaded a couple lists.  When I view files, I see 2 files for each video: a video & audio file.  The video file doesn't have the extension I selected in Options and when opened (with VLC) it doesn't have audio.  Have  I missed a setting that combines these files?

![Screenshot 2023-08-06 221152](https://github.com/shaked6540/YoutubePlaylistDownloader/assets/141598575/543d470e-84f2-4c89-a57c-d823437fb7c8)

![Screenshot 2023-08-06 221326](https://github.com/shaked6540/YoutubePlaylistDownloader/assets/141598575/33fd4aa9-710e-46da-9158-9c8c7be0a689)

**Comment by shaked6540** [comment]
This really shouldn't happen, do you have ffmpeg installed? If you used the installer it would install it for you, but if you built it manually then you need a copy of ffmpeg in the installation folder. 

In any case the program is never meant to copy these partial files to the output directory, I have no idea what's going on. Are you sure you're not viewing the temp folder in the screenshot?

**Comment by fluger12** [comment]
I used the installer and I saw ffmpeg during the installer screen. I also see ffmpeg in the app directory. 
![Screenshot 2023-08-08 064422](https://github.com/shaked6540/YoutubePlaylistDownloader/assets/141598575/8b331df3-ecf8-4355-a99b-48f5760ae88c)

I tried applying an output directory in options but it didn't seem to work. I had to search for the videos when the app was done with download.  It's possible they were in the temp folder and I moved them to the screenshot directory. 

I tried again with another playlist and updated options.  It worked this time.   Does the app download everything in the playlist and then combine files when queue is empty?  If so, I might have moved the files before app was able to do that.  Although I would have to have been pretty quick for at least one not to have been completed.

**Comment by shaked6540** [comment]
This is normal then, your user account doesn't have permission to write files into the save destination. Try running the program as administrator and see if it works
